### PR TITLE
 fix(module:18n): updating pt_PT translations

### PR DESCRIPTION
**Pull request by drackp2m**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

Add and update translations for pt_PT

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[x] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: components/i18n/languages/pt_PT.ts [hunk 1]
```diff
@@ -45,20 +55,28 @@ export default {
       previousDecade: 'Última década',
       nextDecade: 'Próxima década',
       previousCentury: 'Último século',
-      nextCentury: 'Próximo século',
-      placeholder: 'Data',
-      rangePlaceholder: ['Data inicial', 'Data final'],
-      monthFormat: 'MMMM'
```
Comment (wenqi73) [inline components/i18n/languages/pt_PT.ts:51]: Why delete these translations of datePicker and timePicker?

**Review by wenqi73 — CHANGES_REQUESTED** [review]

**Comment by drackp2m** [comment]
If you notice I added them in another order. So that the files are consistent with each other and it is easier to compare translations from one file to another. I took care of leaving es_ES, pt_PT and pt_BR in order with respect to en_US.

Sorry for the delay in answering, I have updated my fork with master and everything is still correct. (No new translations have been added or removed in the file `components/i18n/languages/en_US.ts`

**Comment by wenqi73** [comment]
@drackp2m Oh sorry for the misunderstood, but it is better to restore `monthFormat`, even if the en_US does not have it. By the way, remember to rebase master.

**Comment by drackp2m** [comment]
> rebase master

Perfect @wenqi73, I did it locally, and since I saw that there were no conflicts, I didn't upload it, but today I corrected that.

---

One last question then: What language, or what file could I use to know which translation keys are really valid? Chinese maybe?

**Comment by wenqi73** [comment]
@drackp2m Still has build errors, you should rebase the remote master of ng-zorro.

**Comment by wenqi73** [comment]
> One last question then: What language, or what file could I use to know which translation keys are really valid? Chinese maybe?

@drackp2m The en_US is the valid one, since we sync these files from [ant-design](https://github.com/ant-design/ant-design/tree/master/components/locale), you will see some keys are not used in ng-zorro, maybe we are not using these keys correctly.

**Pull request closed without merging**